timeseries/src/autogluon/timeseries/models/gluonts/torch/models.py

```diff
@@ -162,6 +162,7 @@ class DeepARModel(AbstractGluonTSPyTorchModel):
     """
 
     gluonts_estimator_class: Type[GluonTSPyTorchLightningEstimator] = DeepAREstimator
+    default_num_samples: int = 250
```

Comment:
Just to double-check, the increased number here leads to a lower variance of the forecasts, right?

Reply:
by almost a perfect factor of 2.5 👍